aspnetcore/fundamentals/minimal-apis/7.0-samples/todo-group/data/DbContext.cs

```diff
@@ -0,0 +1,14 @@
+using Microsoft.EntityFrameworkCore;
```

Comment:
This should be in `7.0-samples/todo-group/Data/ApplicationDbContext.cs` instead of `7.0-samples/todo-group/data/DbContext.cs`. Notice that I also capitalized `D` in the `Data` folder name.

I wonder if we can give this a better name like `TodoGroupDbContext` to make it very clear that this isn't some sort of built-in type.

Reply:
ok